bin/tictactoe

```diff
@@ -1,3 +1,10 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+require 'pry'
+
+new_game = GamePlaySelection.new
```

Comment:
Prefer to encapsulate CLI logic into a class. For example, the only code in your bin executable should be:

```ruby
#!/usr/bin/env ruby
       
require_relative '../config/environment'

GamePlaySelection.new.call
```

And then you'd have a file: `lib/game_play_selection.rb` with:

```ruby
class GamePlaySelection
  def initialize
  end

  def call
   # Your game logic
  end
end
```

The code in your bin would generally just go in `call`.